datadog_checks_dev/datadog_checks/dev/tooling/e2e/docker.py

```diff
@@ -270,7 +270,7 @@ def detect_agent_version_linux(self):
     def update_check(self):
         command = ['docker', 'exec', self.container_name]
         command.extend(get_pip_exe(self.python_version, platform=self.container_platform))
-        command.extend(('install', '-e', f'{self.check_mount_dir}[deps]'))
+        command.extend(('install', '--no-input', '-e', f'{self.check_mount_dir}[deps]'))
```

Comment:
This has no effect afaict

Reply:
Any other suggestions as to this might be returning retcode 129?

Reply:
No, but install can never prompt. `--no-input` just happens to be a global option like verbosity, see `pip --help`